app/models/scorebook.rb

```diff
@@ -0,0 +1,97 @@
+class Scorebook
```

Comment:
@kriskelly Since the models folder is already pretty crowded, it'd be nice to categorize ruby classes into other folders where we can, keeping models/ for ActiveRecord. Apparently a 'queries' folder is something people do (http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)